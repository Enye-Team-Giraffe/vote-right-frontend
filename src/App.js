import React from 'react';
import Login from './components/login/components';

function App() {
    return (
        <Login />
    );
}
export default App;
