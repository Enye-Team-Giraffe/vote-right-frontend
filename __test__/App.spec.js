import React from 'react';
import { mount } from './enzyme';
import App from '../src/App';

describe('App', () => {
    it('should render my component', () => {
        const wrapper = mount(<App />);
        expect(wrapper).toMatchSnapshot();
    });
});
