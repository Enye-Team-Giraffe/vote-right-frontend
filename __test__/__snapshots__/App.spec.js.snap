// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render my component 1`] = `ReactWrapper {}`;
